src/acceleration/AitkenAcceleration.cpp

```diff
@@ -52,6 +53,12 @@ void AitkenAcceleration::performAcceleration(
 {
   PRECICE_TRACE();
 
+  for (const auto &data : cplData | boost::adaptors::map_values) {
+    if (data->exchangeSubsteps()) {
+      PRECICE_ERROR("Acceleration scheme does not support subcycling. Please pick a different acceleration scheme or set substeps=\"false\" in the exchange tag of data \"{}\".", data->getDataName());
```

Comment:
Isn't subcycling here the wrong term?
How I would use the term, subcycling is supported, but waveform iteration is not.
Maybe we could use sth like

> Aitken acceleration does not yet support using data from all substeps.

"Pick a different acceleration scheme" could also be dangerous. User will start testing all methods. I can already imagine the forum post. I would just drop this sentence. 

Why not triggering the error in intialize already?

Reply:
Triggering error in initialize already and updated the error message.